post_installation_configuration/post-install-storage-configuration.adoc

```diff
@@ -20,8 +20,13 @@ endif::[]
 
 toc::[]
 
-After installing {product-title}, you can further expand and customize your
-cluster to your requirements, including storage configuration.
+After installing {product-title}, you can further expand and customize your cluster to your requirements, including storage configuration.
```

Comment:
Does this module belong in here?  Did you miss it in [your other PR](https://github.com/openshift/openshift-docs/pull/79372/files)?

Reply:
Since I was removing [this](https://github.com/openshift/openshift-docs/pull/79400/files#diff-972357080f7d8daa8904cb2f48eaaeb4dcb4bbd24a6f35c3a127628724df30afL59-L72) content to make the landing page less crowded, I figured it made sense to move it in this PR since the other PR to clean up storage is already merged. I can do a separate PR for this if necessary, but it seemed like a small enough change that it made sense to just move it in this one.